sdk/monitor/azquery/utils_test.go

```diff
@@ -61,10 +66,23 @@ func TestMain(m *testing.M) {
 	if recording.GetRecordMode() == recording.PlaybackMode {
 		credential = &FakeCredential{}
 	} else {
-		credential, err = azidentity.NewDefaultAzureCredential(nil)
+		tenantID := lookupEnvVar("AZQUERY_TENANT_ID")
+		clientID := lookupEnvVar("AZQUERY_CLIENT_ID")
+		secret := lookupEnvVar("AZQUERY_CLIENT_SECRET")
+		credential, err = azidentity.NewClientSecretCredential(tenantID, clientID, secret, nil)
 		if err != nil {
 			panic(err)
 		}
+
+		if cloudEnv, ok := os.LookupEnv("AZQUERY_ENVIRONMENT"); ok {
+			if cloudEnv == "AzureUSGovernment" {
```

Comment:
You should use [`strings.EqualFold`](https://pkg.go.dev/strings#EqualFold) for a case-insensitive comparison.

Reply:
fixed in commit [d2eafa6](https://github.com/Azure/azure-sdk-for-go/pull/19462/commits/d2eafa65677cd3eeccdfa1f740ecf1ff6bd959f9)